README.md

```diff
@@ -0,0 +1,200 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="soyMarco.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Metric Web App <a name="about-project"></a>
+
+<div align="center">
+
+  <img src="stock_stats.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+</div>
+
+**Metric Web App** is a project to show API data on a SPA.
+![Video](https://cdn.loom.com/sessions/thumbnails/dec4f57c759b400e8c4e7f14cb1df71e-with-play.gif)
```

Comment:
- [ ] Your video for the project is truly exceptional :+1:, showcasing your remarkable skills. Nevertheless, it is essential to include a highlight in the video that focuses on a specific portion of the code which you personally find intriguing or take pride in creating. This requirement serves the purpose of highlighting your expertise and emphasizing the significance of a particular code segment within the overall project.

Reply:
Thank you for your suggestion, I will apply that now...